depends/packages/miniupnpc.mk

```diff
@@ -12,6 +12,10 @@ $(package)_build_opts_mingw32=-f Makefile.mingw
 $(package)_build_env+=CFLAGS="$($(package)_cflags) $($(package)_cppflags)" AR="$($(package)_ar)"
 endef
 
+# sed is used here to replace the host and version that will end up as part of
+# the miniupnpc User-Agent string, that will ultimately end up in bitcoind. i.e
+# before: User-Agent: Debian/10, UPnP/1.1, MiniUPnPc/2.0
+# after : User-Agent: x86_64-pc-linux-gnu, UPnP/1.1, MiniUPnPc/2.0.20180203
```

Comment:
unrelated to this change but: why are we inserting the architecture in here, seems like something not necessary to reveal

Reply:
I agree. Why make life easier for an attacker? :)

Reply:
My assumption is that it was being inserted to make depends more deterministic. However I think we can improve this, as there's no reason to reveal more of the version information either. Could leave the version as is `/2.0` and either drop the architecture, or insert something else?

Reply:
I guess `UPnP/1.1`, the version of the standard used, not the version of the software, is the only relevant thing here. The rest just aids exploiters if there's another remotely exploitable UPnP bug.